pkg/webhooks/machine_webhook.go

```diff
@@ -111,6 +147,18 @@ const (
 	minVSphereMemoryMiB = 2048
 	// https://docs.openshift.com/container-platform/4.1/installing/installing_vsphere/installing-vsphere.html#minimum-resource-requirements_installing-vsphere
 	minVSphereDiskGiB = 120
+
+	// PowerVS Defaults
+	defaultPowerVSCredentialsSecret = "powervs-credentials"
+	defaultPowerVSSysType           = "s922"
+	defaultPowerVSProcType          = "Shared"
+	defaultPowerVSProcessor         = "0.5"
+	defaultPowerVSMemory            = 32
+	powerVSServiceInstance          = "serviceInstance"
+	powerVSNetwork                  = "network"
+	powerVSImage                    = "image"
+	powerVSSystemTypee880           = "e880"
+	powerVSSystemTypee980           = "e980"
```

Comment:
Nit, I would Capitalise the E to make it clear it's something new
```suggestion
	powerVSSystemTypeE880           = "e880"
	powerVSSystemTypeE980           = "e980"
```

Reply:
This comment still needs resolving

Reply:
I missed it earlier, Updated it now